tensorflow/core/graph/gpu_fusion_pass.cc

```diff
@@ -456,27 +444,39 @@ bool ROCmFusionPass::RunPass(Graph* graph) {
 }
 
 void ROCmFusionPass::InitializeFusions(
-    std::vector<std::unique_ptr<ROCmFusionOpBase> >& fusions, Graph* g,
-    std::set<const Node*>& fused_nodes) {
-  fusions.emplace_back(
-      new ROCmFusionOpConvolutionBiasBatchNormActivation(g, fused_nodes));
-  fusions.emplace_back(
-      new ROCmFusionOpConvolutionBiasActivation(g, fused_nodes));
-  fusions.emplace_back(new ROCmFusionOpBatchNormActivation(g, fused_nodes));
-  fusions.emplace_back(new ROCmFusionOpAddRelu(g, fused_nodes));
-  fusions.emplace_back(new ROCmFusionOpAddNReluGrad(g, fused_nodes));
+    std::vector<std::unique_ptr<ROCmFusionOpBase> >& fusions, Graph* g) {
+  if (!getenv("TF_ROCM_FUSION_DISABLE_CBNA")) {
```

Comment:
It seems in TF there are utility functions `ReadXXXFromEnvVar`. Could you help use that? Perhaps `ReadBoolFromEnvVar` would be a good choice.

Reply:
make sense...will make that change in the next commit

Reply:
the test is a success. once you make the change I'll take it.